interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,7 +3,7 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <gene-search-form></gene-search-form>
+    <gene-search-panel organism="organism"></gene-search-panel>
```

Comment:
Yep, I think (as @dongbohu said) you can specify the location & name of the template file in whatever format (but build.config.js will care about only including template files that end in 'tpl.html', as we have it currently configured). I think the only other thing that needs to follow the strict hyphens <-> camelCase connection is the name of the directive itself (camelCase) and the name of the element it will replace in the html template (hyphens). I am totally fine if we decide to standardize on a specific format for template names though.